addons/website/static/src/snippets/s_dynamic_filter_carousel/000.scss

```diff
@@ -0,0 +1,27 @@
+.s_dynamic_filter_carousel {
+  .o_carousel_dynamic_filter_control {
+    top: 33.33333333%;
```

Comment:
can we ad another 3 at the end ;) 
should probably use power of saas 1/3, and variable $o-enterprise-primary-color for color e.g.

but do we really need this css ? 
Can't we use standard carousel layout?
We avoid magic number, 12/8/12 seems louchh, is it really needed ? Seems more to be beautiful with demo data and partner name that really a justified use case